x-pack/plugins/security_solution/server/endpoint/services/actions/action_list.test.ts

```diff
@@ -316,14 +316,7 @@ describe('When using `getActionList()', () => {
               errors: undefined,
             },
           },
-          outputs: {
-            'agent-a': {
-              content: {
-                code: 'ra_scan_success_done',
-              },
-              type: 'json',
-            },
-          },
+          outputs: {},
```

Comment:
I am not sure why this was changed, anybody has more knowledge about this suite?